tmol/tests/data/term_baselines/HBondEnergyTerm/test_whole_pose_scoring_10.yaml

```diff
@@ -0,0 +1,11 @@
+term0:
+  pose0: -55.675621032714844
+  pose1: -55.675621032714844
+  pose2: -55.67561721801758
+  pose3: -55.675621032714844
+  pose4: -55.675621032714844
+  pose5: -55.675621032714844
+  pose6: -55.675621032714844
+  pose7: -55.67562484741211
+  pose8: -55.675621032714844
+  pose9: -55.67561721801758
```

Comment:
Yeah, should be.

It's a little weird to have it displayed like this, but I think it's worth keeping this way, since it lets you abstract the saved/computed results as just complete tensors without having to know anything about their structure.